treasury/src/constants/constants.ts

```diff
@@ -23,8 +23,8 @@ interface Announcement {
 }
 
 export const ANNOUNCEMENT: Announcement | undefined = {
-  color: "#E84142",
-  message: "AVAX vaults have launched.",
-  linkText: "Switch to Avalanche",
+  color: "#FFFFFF",
+  message: "Aurora vaults have launched.",
```

Comment:
We are launching with NEAR first

Reply:
Sure, although this maybe a bit confusing to clients because we are launching wrapped NEAR on Aurora.